meta/helper/solrObjects.ts

```diff
@@ -1,3 +1,4 @@
+import { c } from "nuqs/dist/serializer-C_l8WgvO";
```

Comment:
Was this an intentional import? I don't see it used and maybe it was auto-added by your IDE?